vue-components/src/styles/mixins/Label.scss

```diff
@@ -5,8 +5,8 @@
     font-weight: $wikit-Label-font-weight;
     line-height: $wikit-Label-line-height;
     display: $displayType;
-     
-    @if $displayType == block {
+
+    @if $displayType == block or $displayType == inline-flex {
```

Comment:
Why do we need this change?

Reply:
we don't. I forgot to remove that from a previous commit. 
removed now